### DISCUSSION TITLE
Unable to validate pathname once a form was sent

### DISCUSSION THREAD

**Discussion by osoriosara98** [cypress-io/cypress#26405] [Questions and Help]
I'm trying to validate that the user was redirected to a thank you page after sending a form

I used this line after completing the form and clicking on submit.

![1](https://user-images.githubusercontent.com/102264353/229563960-39a8dda2-629f-4708-a44a-62c54dcdc317.png)

cy.location('pathname').should('eq', '/thank-you/')

**Comment by jjhelguero** [comment]
A form submission should trigger requests sent by the app. Have you tried intercepting and waiting on those before the page redirects? Alternatively, you can check the page for any loading icons or new page elements.

**Reply by osoriosara98** [reply]
yeah, I tried that already. I even added a long waiting time but it didn't work
